server/pages/index.js

```diff
@@ -359,7 +361,13 @@ module.exports = function() {
 			// the user is probably logged in
 			renderLoggedInRoute( req, res );
 		} else {
-			renderLoggedOutRoute( req, res );
+			const LayoutLoggedOutDesign = require( 'layout/logged-out-design' ),
+				LayoutLoggedOutDesignFactory = React.createFactory( LayoutLoggedOutDesign ),
```

Comment:
Though `LayoutLoggedOutDesign` is expected to be found in the `require` cache, the `createFactory` call likely isn't cached. Should we move these to the module's top level?